lib/Reader/reader.cpp

```diff
@@ -2568,6 +2569,13 @@ void ReaderBase::fgBuildPhase1(FlowGraphNode *Block, uint8_t *ILInput,
       HasLocAlloc = true;
       break;
 
+    case ReaderBaseNS::CEE_LDLOCA:
+    case ReaderBaseNS::CEE_LDLOCA_S:
+    case ReaderBaseNS::CEE_LDARGA:
+    case ReaderBaseNS::CEE_LDARGA_S:
```

Comment:
The [Lang Ref for the tail annotation](http://llvm.org/docs/LangRef.html#call-instruction) says
> Both markers imply that the callee does not access allocas or varargs from the caller

so does this need to include `CEE_ARGLIST`?